tests/SideBySide/SideBySide.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
  <PropertyGroup Condition=" '$(Configuration)' != 'Baseline' ">
-    <TargetFrameworks>netcoreapp1.1.1;net462</TargetFrameworks>
```

Comment:
Any particular reason to change `net462` -> `net461` and `netcoreapp1.1.1` -> `netcoreapp1.1.2` here? (Mostly just curious.)